src/components/ArticlesTable/ArticlesTable.js

```diff
@@ -91,11 +91,11 @@ export class ArticlesTable extends Component {
                     >
                       <MenuItem
                         disabled
-                        primaryText={`Saved: ${moment(item.get('timestamp')).format('lll')}`}
+                        primaryText={`Saved: ${moment.utc(item.get('timestamp')).local().format('lll')}`}
```

Comment:
Well, re-ready your own link:
> a programmer should not add functionality until deemed necessary.

Emphasis on _functionality_; it does not apply for refactoring and code cleaning. Indeed,
> It is meant to be used in combination with several other practices, such as continuous refactoring, ...